cli/src/merge_tools/mod.rs

```diff
@@ -431,6 +450,7 @@ mod tests {
                 name,
                 &settings,
                 GitIgnoreFile::empty(),
+                Some(GitAttributesFile::empty()),
```

Comment:
I think this (and below) can just be `None` unless you're planning on adding tests around this.